### PR TITLE
Update 05-exprs-qc.Rmd

### PR DESCRIPTION
**Pull request by tallulandrews**

### CODE REVIEW

File: 05-exprs-qc.Rmd [hunk 1]
```diff
@@ -396,6 +337,40 @@ limma::vennDiagram(venn.diag,
                    circle.col = c("magenta", "blue", "green"))
 ```
 
+## Gene filtering
```
Comment (wikiselev) [inline 05-exprs-qc.Rmd]: I think Gene filtering should be before Cell filtering... Why did you move it to the end?

Comment (tallulandrews) [inline 05-exprs-qc.Rmd]: Gene filtering has to be after cell filtering because I've had several
cases where genes are only detected in the cells that are later removed
because they are low quality, thus if you do gene filtering before cell
filtering you can end up with rows that are all zeros in the final
expression matrix which screws up some methods.

Tallulah
